README.md

```diff
@@ -164,6 +165,19 @@ Any customer projects accessible by end users need to support whatever browsers
 
 When adding support for additional browsers infers very little overhead, we should support as broadly as possible. The two latest versions of all major browsers is the industry standard. No version of Internet Explorer is formally supported.
 
+#### System identification
+
+All outgoing HTTP requests should identify the caller via a properly formatted `User-Agent` header. This header should at minimum describe the name, version and environment of the service. All Aptoma services should prefix the system name with `aptoma-`. Additionally, services may add arbitrary data as key-value pairs in parentheses. This format is specified by [RFC9110](https://www.rfc-editor.org/rfc/rfc9110#field.user-agent).
+
+Examples:
+
+```
+User-Agent: aptoma-drpublish/1.2.3 (env:production)
```

Comment:
Any thoughts on `aptoma-drpublish/1.2.3` vs `aptoma drpublish/1.2.3`. Both are valid. I'm thinking the main reason would be to make it easy to either filter out all aptoma or all dpublish request, but I suppose partial string matching will work equally fine with either option.

Reply:
If both are valid I'd prefer having aptoma stand alone instead of using it as a prefix. I found the spec to be a bit unclear on that, or if you could have an identifier without a version.

Reply:
> By convention, the product identifiers are listed in decreasing order of their significance for identifying the user agent software. Each product identifier consists of a name and optional version.

Identifier sounds optional to me.

Reply:
Edit: Version sounds optional.